### PR TITLE
Check if any packages are available before calling pip install

### PR DESCRIPTION
**Pull request by smac89**
Fix error when no packages are available for update

### CODE REVIEW

File: bin/pyenv-pip-update [hunk 1]
```diff
@@ -27,7 +27,11 @@ then # If no argument is given, update libraries using pip
             conda update --all
         elif pip >/dev/null 2>&1
         then
-            pip list --outdated | awk '{if ($1 != "Package" && $1 !~ /^-+$/) print $1}' | xargs pip install -U
+            stale_packages="$(pip --no-python-version-warning list --format=freeze --outdated | awk -F '==' '{printf "%s ", $1}')"
```
Comment (massongit) [inline bin/pyenv-pip-update]: You need to decide whether to use `--no-python-version-warning` depending on the version of pip becouse this option isn't available before pip 20.0.

Comment (smac89) [reply bin/pyenv-pip-update]: Fixed

**Comment by smac89** [comment]
I was thinking to add a check for system python, and avoid updating using pip because most of the time those require root permissions. What do you think?

**Comment by massongit** [comment]
I think that it is better to print the command needed and exit if the system python need upgrade packages.
Because I think that many people don't want to update packages in system python.

**Comment by massongit** [comment]
Issue cleated: https://github.com/massongit/pyenv-pip-update/issues/5

**Comment by massongit** [comment]
I merge it.

**Pull request merged**